common/src/main/scala/org/apache/celeborn/common/RssConf.scala

```diff
@@ -713,17 +903,92 @@ object RssConf extends Logging {
     conf.getInt(key, HA_MASTER_NODE_RATIS_PORT.defaultValue.get)
   }
 
-  def workerReplicateNumThreads(conf: RssConf): Int = {
-    conf.getInt("rss.worker.replicate.numThreads", 64)
-  }
+  def haMasterRatisRpcType(conf: RssConf): String = conf.get(HA_MASTER_RATIS_RPC_TYPE)
+  def haMasterRatisStorageDir(conf: RssConf): String = conf.get(HA_MASTER_RATIS_STORAGE_DIR)
 
-  def workerAsyncCommitFileThreads(conf: RssConf): Int = {
-    conf.getInt("rss.worker.asyncCommitFiles.numThreads", 32)
-  }
+  def haMasterRatisLogSegmentSizeMax(conf: RssConf): Long =
+    conf.get(HA_MASTER_RATIS_LOG_SEGMENT_SIZE_MAX)
 
-  def workerFlushBufferSize(conf: RssConf): Long = {
-    conf.getSizeAsBytes("rss.worker.flush.buffer.size", "256k")
-  }
+  def haMasterRatisLogPreallocatedSize(conf: RssConf): Long =
+    conf.get(HA_MASTER_RATIS_LOG_PREALLOCATED_SIZE)
+
+  def haMasterRatisLogAppenderQueueNumElements(conf: RssConf): Int =
+    conf.get(HA_MASTER_RATIS_LOG_APPENDER_QUEUE_NUM_ELEMENTS)
+
+  def haMasterRatisLogAppenderQueueBytesLimit(conf: RssConf): Long =
+    conf.get(HA_MASTER_RATIS_LOG_APPENDER_QUEUE_BYTE_LIMIT)
+
+  def haMasterRatisLogPurgeGap(conf: RssConf): Int = conf.get(HA_MASTER_RATIS_LOG_PURGE_GAP)
+
+  def haMasterRatisRpcRequestTimeout(conf: RssConf): Long =
+    conf.get(HA_MASTER_RATIS_RPC_REQUEST_TIMEOUT)
+
+  def haMasterRatisRetryCacheExpiryTime(conf: RssConf): Long =
+    conf.get(HA_MASTER_RATIS_SERVER_RETRY_CACHE_EXPIRY_TIME)
+
+  def haMasterRatisRpcTimeoutMin(conf: RssConf): Long = conf.get(HA_MASTER_RATIS_RPC_TIMEOUT_MIN)
+
+  def haMasterRatisRpcTimeoutMax(conf: RssConf): Long = conf.get(HA_MASTER_RATIS_RPC_TIMEOUT_MAX)
+
+  def haMasterRatisNotificationNoLeaderTimeout(conf: RssConf): Long =
+    conf.get(HA_MASTER_RATIS_NOTIFICATION_NO_LEADER_TIMEOUT)
+
+  def haMasterRatisRpcSlownessTimeout(conf: RssConf): Long =
+    conf.get(HA_MASTER_RATIS_RPC_SLOWNESS_TIMEOUT)
+
+  def haMasterRatisRoleCheckInterval(conf: RssConf): Long =
+    conf.get(HA_MASTER_RATIS_ROLE_CHECK_INTERVAL)
+
+  def haMasterRatisSnapshotAutoTriggerEnabled(conf: RssConf): Boolean =
+    conf.get(HA_MASTER_RATIS_SNAPSHOT_AUTO_TRIGGER_ENABLED)
+
+  def haMasterRatisSnapshotAutoTriggerThreshold(conf: RssConf): Long =
+    conf.get(HA_MASTER_RATIS_SNAPSHOT_AUTO_TRIGGER_THRESHOLD)
+  def haMasterRatisSnapshotRetentionFileNum(conf: RssConf): Int =
+    conf.get(HA_MASTER_RATIS_SNAPSHOT_RETENTION_FILE_NUM)
+
+  val WORKER_REPLICATE_THREADS: ConfigEntry[Int] =
+    buildConf("celeborn.worker.replicate.threads")
```

Comment:
I have searched the spark codebase, it has two styles to represent thread numbers, "threads" and "numThreads", I think we should unify the representation in celeborn, personally, I lean toward the shorter.

Reply:
+1